pan-domain-auth-hmac/src/test/scala/com/gu/pandomainauth/PanDomainAuthSettingsRefresherTest.scala

```diff
@@ -0,0 +1,34 @@
+package com.gu.pandomainauth
+
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.s3.AmazonS3ClientBuilder
+import com.gu.pandomainauth.model.AuthenticatedUser
+import com.gu.pandomainauth.service.CookieUtils
+import com.gu.pandomainauth.service.CryptoConf.SigningAndVerification
+import org.scalatest.freespec.AnyFreeSpec
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.{EitherValues, OptionValues}
+
+import java.time.Instant
+import java.time.temporal.ChronoUnit.HOURS
+
+class PanDomainAuthSettingsRefresherTest extends AnyFreeSpec with Matchers with EitherValues with OptionValues {
+
+  "Give an expired cookie" in {
+    val myRealCookie = ""// "Paste in your real cookie value here - from the 'gutoolsAuth-assym' cookie"
+
+    val s3Client =
+      AmazonS3ClientBuilder.standard().withRegion(Regions.EU_WEST_1)
+        .withCredentials(new ProfileCredentialsProvider("workflow")).build()
+
+    val signingAndVerification: SigningAndVerification =
+      PanDomainAuthSettingsRefresher("gutools.co.uk", "testing",
```

Comment:
Joe, make sure you match `gutools.co.uk` to the domain you got your cookie from, eg it may need to be `code.dev-gutools.co.uk`